locust_tasks/locustfile.py

```diff
@@ -334,7 +474,21 @@ def identify_page(id, task, resp):
     # Identification failed
     failure_message = f'Failed to identify page. Status={resp.status_code}.'
     report_failure(id, resp, task, failure_message, clean_text(page_content))
-    
+
+"""
+Returns the address that corresponds to the uprn. This can then be used to select the correct address from the page.
```

Comment:
The description is misleading.
It's extracting the html 'value' for a radio button of the target address. 
The value just so happens to contain human readable text for the address but the causal reader wouldn't know that, so calling an address will likely confuse them.

Reply:
I'll change it